site/docs/v1/tech/guides/breached-password-detection.adoc

```diff
@@ -0,0 +1,274 @@
+
+Passwords will be checked any time they are changed, such as:
+
+* at user registration
+* when a password change initiated by an end user
+* a password changes made by an administrative user
+
+You can see below that an administrative user will receive an error message should they attempt to change an end user's password to a value known to be breached:
+
+image::guides/breached-password-detection/admin-user-attempting-to-use-breached-password.png[Error message when an administrative user tries to use a compromised password.,width=700,role=box-shadow margin-right]
+
+However, if an administrative user changes any other user attribute without modifying the password, the current password won't be checked.
+
+Password changes, whether triggered by a user interacting with the OAuth registration pages or an application using the API, will cause the password to be checked against the latest corpus of compromised passwords. If there is a match, an error will be returned.
+
+== Configuration Options
+
+While the default settings are good, you may want to configure this feature further. Here are the configuration options:
+
+image::guides/breached-password-detection/breached-password-configuration-options.png[Breached password detection configuration options.,width=700,role=box-shadow margin-right]
+
+Each of these configuration options can be updated using either the administrative interface or by calling the link:../apis/tenants[Tenant API].
+
+=== Match Mode
+
+When the system is seeing if credentials have been leacked, there are a number of ways to attempt to match them with the datasets. Assume you have a user with the username `richard@piedpiper.com` and the password `password!`.
+
+One option is that the system could match on `password!` only. This means that if a user tries to change their password to `password!`, the system would see if that string had ever been leaked. This means that if any other user had ever used that password and it had been compromised, FusionAuth would disallow its use.
+
+Another way of matching would be to see if there are any sub-email accounts which are associated with that password. These are also called aliases by some email providers. For instance, `richard+test@piedpiper.com` is an alias of `richard@piedpiper.com`. The system could check multiple aliases, such as `richard+test@piedpiper.com`, `richard+foo@piedpiper.com` and so on. All of these addresses would be paired with the provided password and checked against the datasets.
+
+You could also narrow your scope further and match on only the username/password pair. The system can examine if `richard@piedpiper.com` and `password!` appeared together in the leaked passwords.
+
+Finally, the system could check if the string `password!` is a commonly compromised password. If it is prevalent in breached password lists, it is a good password to avoid.
```

Comment:
Functionally, this seems like the same as the first example with `password!`. Unless we want to discuss that we short circuit the breach logic with this common set, can we omit this part? 

Maybe we have to describe it because we mention in the match mode config already?

Reply:
I am mentioning it for thoroughness, because it is mentioned if you set the match mode to `Low`. But I just removed it and think we can get by just mentioning 'commonly compromised password' without defining that further.